app/views/components/fields/field_multiple_values.scala.html

```diff
@@ -27,7 +27,9 @@
     labelClass: Option[String] = None,
     input: Html,
     includeAddButton: Boolean = true,
-    valuePrefix: Option[String] = None
+    valuePrefix: Option[String] = None,
+    addHint: String = "site.add",
+    removeHint: String = "site.remove.hint"
 )(implicit messages: Messages)
```

Comment:
Nit: This component's arguments list grew signifficantly. We might need to think about either splitting the component (as it might be doing too much things), or to somehow reduce the amount of parameters.

Reply:
It's a sign that it had lots of configuration options.  Not sure what the solution is though.  Either the component is re-usable or we create multiple components for use on specific forms/pages

Reply:
So maybe we could wrap all configuration params into one class and pass it then?